src/kernel.cpp

```diff
@@ -109,5 +110,6 @@ extern "C" void kmain()
 	terminal_writestring("welcome to my first operating system!");
 	terminal_writestring("welcome to my first operating system!");
 	terminal_writestring("welcome to my first operating system!");
+	fb_move_cursor(81);
```

Comment:
Do we want to keep this in? Maybe we can do an initial `fb_move_cursor(0)` at the start of `kmain`!

Reply:
Yeah, let's keep this in, since fb_move_cursor(0) is how it is at the beginning.